app/gui2/src/components/GraphEditor/widgets/WidgetApplication.vue

```diff
@@ -48,7 +48,10 @@ export const widgetDefinition = defineWidget(ArgumentApplicationKey, {
       <NodeWidget :input="WidgetInput.FromAst(application.infixOperator)" />
     </div>
     <Transition name="collapse-argument">
-      <div v-if="tree.extended || !application.argument.hideByDefault" class="argument">
+      <div
+        v-if="tree.extended || !application.argument.hideByDefault"
+        class="argument overridableClipState"
+      >
```

Comment:
Is it the only place which may clip drop-down? Could node clip the leftmost drop-down?

Reply:
This prevents the clipping required by the hidden-placeholders animation from interfering with the dropdown. I don't think we have any other animations that could clip the dropdown. If any widget is enabling clipping on its children statically, that would be a separate problem with a different solution (the widget must not do that).